src/gazebo_mavlink_interface.cpp

```diff
@@ -549,6 +590,25 @@ void GazeboMavlinkInterface::send_mavlink_message(const mavlink_message_t *messa
   }
 }
 
+void GazeboMavlinkInterface::forward_mavlink_message(const mavlink_message_t *message)
+{
+  uint8_t buffer[MAVLINK_MAX_PACKET_LEN];
+  int packetlen = mavlink_msg_to_send_buffer(buffer, message);
+
+  ssize_t len = sendto(qgc_socket_fd_, buffer, packetlen, 0, (struct sockaddr *)&remote_qgc_addr_, remote_qgc_addr_len_);
```

Comment:
What if the QGC socket is not initialized?

Reply:
It should be because we abort otherwise:
https://github.com/PX4/sitl_gazebo/pull/316/files/722632f16e06e68480bca36fa82a80d327e1196b..97046d68d486120c8f1b5a1d5bfddc5d896f49c1#diff-af95b4649436828244349b1766ab5c46R394

Reply:
But that is conditioned on `if (serial_enabled_) {`

Reply:
Has this been addressed?